go.mod

```diff
@@ -3,8 +3,19 @@ module github.com/nats-io/nats.go
 go 1.18
 
 require (
+	github.com/golang/protobuf v1.5.2
+	github.com/nats-io/nats-server/v2 v2.9.11
 	github.com/nats-io/nkeys v0.3.0
 	github.com/nats-io/nuid v1.0.1
+	golang.org/x/exp v0.0.0-20230108222341-4b8118a2686a
```

Comment:
there is a different `go_test.mod` where you can add test dependencies and can be called via `go test -modfile=go_test.mod`

Reply:
fixed